src/client/App.js

```diff
@@ -1,26 +1,33 @@
 import React from "react";
 import { BrowserRouter as Router, Route, Switch } from "react-router-dom";
-import MealList from "./components/components/MealList";
+import { MealProvider } from "./components/MealContext";
+import MealList from "./components/Meals/MealList";
 import TestComponent from "./components/TestComponent/TestComponent";
-import MealList from "./components/TestComponent/MealList";
+import { MealContext } from "./components/MealContext";
+import Home from "./components/Home/Home";
+import Review from "./components/Reviews/Review";
+import MealDetails from "./components/Reservations/MealDetails"
+import Footer from "./components/Footer/Footer";
+import Navbar from "./components/Navbar/Navbar";
 
 function App() {
+  console.log("randering app");
   return (
     <Router>
-      <Route exact path="/meals">
-        <p>Meal List</p>
-        <MealList />
-      </Route>
       <Route exact path="/lol">
         <p>lol</p>
       </Route>
-      <Route exact path="/meals">
-        <p>Meal List</p>
-        <MealList/>
-      </Route>
+      <Navbar/>
+      <MealProvider>
+        <Route exact path="/" component={Home}/>
+        <Route exact path="/meals" component={MealList}/>
+        <Route path="/meals/:id" component={MealDetails} />
+      </MealProvider>
+      <Route exact path="/reviews" component={Review} />
```

Comment:
reviews or Review? 🤔

Reply:
yeah it should be Reviews but its because it was bit confusing for me with names in database and in nodejs and here so...